app/freertos-sample/CMakeLists.txt

```diff
@@ -32,14 +32,8 @@ target_compile_definitions(
 )
 
 target_link_libraries(${PROJECT_NAME}
-    "-Wl,--start-group"
-    c
-    gcc
-    m
-    metal
-    kernel
     freertos
-    "-Wl,--end-group"
+    kernel
```

Comment:
Is the kernel project the only one that is really needed here?  I think freertos is already included in kernel library.